docs/sphinx/_toc.yml.in

```diff
@@ -11,24 +11,10 @@ subtrees:
 
 - caption: Installation
   entries:
-  - file: install/windows/install-quick.md
-    title: Quick start (Windows)
-  - file: install/windows/install.md
-    title: Windows install guide
-    subtrees:
-    - entries:
-      - file: install/windows/windows-app-deployment-guidelines.md
-        title: Application deployment guidelines
-  - file: install/docker.md
-    title: ROCm Docker containers
-  - file: install/pytorch-install.md
-    title: PyTorch for ROCm
-  - file: install/tensorflow-install.md
-    title: Tensorflow for ROCm
-  - file: install/magma-install.md
-    title: MAGMA for ROCm
-  - file: install/spack-intro.md
-    title: ROCm & Spack
+  - file: ${project:linux-install}
```

Comment:
The project was named "linux". 

Do we want to change it to "linux-install"?

Reply:
I have merged the name change on rocm-docs-core?